project/Doc.scala

```diff
@@ -74,7 +74,7 @@ object Scaladoc extends AutoPlugin {
       "-doc-version",
       ver,
       "-doc-canonical-base-url",
-      "https://doc.akka.io/api/akka/current/")
```

Comment:
A couple of lines above this - could you replace this?

```
      "-doc-title",
      "Akka",
```
with
```
      "-doc-title",
      "Apache Pekko",
```

Reply:
Done